### PR TITLE
 [12.0][MIG] packaging_uom

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by rousseldenis** [comment]
Changed repository after closing this https://github.com/OCA/stock-logistics-warehouse/pull/600 to make it more coherent.

File: packaging_uom/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Packaging UOM",
+    "version": "12.0.1.0.0",
+    "author": 'ACSONE SA/NV, '
+              'Odoo Community Association (OCA)',
+    "category": "Warehouse",
+    "development_status": "Production/Stable",
+    "maintainers": ["rousseldenis"],
+    "website": "http://github.com/stock-logistics-warehouse",
```
Comment (tbaden) [inline packaging_uom/__manifest__.py]: https://github.com/stock-logistics-warehouse would be nice.

**Review by tbaden — APPROVED** [review]
thanks for the work 👍 
I did a code review: LGTM
maybe change url to https and if you like use original ids in xml records, but these are optional.

**Review by lmignon — APPROVED** [review]

File: packaging_uom/__manifest__.py (hunk 1 above)
Comment (rousseldenis) [reply packaging_uom/__manifest__.py]: Done !

**Comment by rousseldenis** [comment]
@rafaelbn

**Comment by rousseldenis** [comment]
Wait, there is maybe a bug on views level

**Comment by rafaelbn** [comment]
Could we activate packaging option by default when using this module?

**Review by rafaelbn — APPROVED** [review]
Tested functionally in runbot :+1:

**Comment by andres-pcg** [comment]
Tested it in runbot. 👍  
Thanks @rousseldenis

**Comment by rousseldenis** [comment]
So, this one is ready !

**Comment by rousseldenis** [comment]
@OCA/product-maintainers

**Comment by gustavovalverde** [comment]
@pedrobaeza could we have some merge love here?

**Pull request merged**